modules/ROOT/pages/_partials/supported-versions.adoc

```diff
@@ -325,7 +341,7 @@ Please plan to migrate your apps to use an appropriate alternative version.
 | OS|Version|Type
 
 |RedHat
-|9
+|8+
```

Comment:
wait what, Debian 8 is supported, but only 9,10 are deprecated?
Does that mean that 8 isn't deprecated?
Or that it was deprecated in a previous release so we don't have to mention it?